src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+import { Tooltip } from './Tooltip'
+
+describe('Tooltip component', () => {
+  it('renders without errors', () => {
+    const { queryByTestId } = render(
+      <Tooltip label="Click me">
+        <button>My Tooltip</button>
+      </Tooltip>
+    )
+    expect(queryByTestId('tooltipWrapper')).toBeInTheDocument()
+    expect(queryByTestId('triggerElement')).toBeInTheDocument()
+    expect(queryByTestId('tooltipBody')).toBeInTheDocument()
+  })
+
+  it('hides tooltip by default', () => {
+    const { getByTestId } = render(
+      <Tooltip label="Click me">
+        <button>My Tooltip</button>
+      </Tooltip>
```

Comment:
Sorry, I didn't see this on my first pass. Looks like the unit tests are still using the old implementation syntax for the tooltip. Can we update these to match how Tooltip is used in storybook now that it implements a button internally?